interface/modules/custom_modules/oe-module-claimrev-connect/public/setup.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+/**
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ *
+ * @author    Brad Sharp <brad.sharp@claimrev.com>
+ * @copyright Copyright (c) 2022 Brad Sharp <brad.sharp@claimrev.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+    require_once "../../../../globals.php";
+
+    $ta = "setup";
+    use OpenEMR\Modules\ClaimRevConnector\ClaimRevModuleSetup;
+
+    $services = ClaimRevModuleSetup::getBackgroundServices();
+if (isset($_POST['deactivateSftp'])) {
+    ClaimRevModuleSetup::deactivateSftpService();
+}
+if (isset($_POST['reactivateSftp'])) {
+    ClaimRevModuleSetup::reactivateSftpService();
+}
+if (isset($_POST['backgroundService'])) {
+    ClaimRevModuleSetup::createBackGroundServices();
+}
+
+$services = ClaimRevModuleSetup::getBackgroundServices();
+
+?>
+<html>
+    <head>
+        <link rel="stylesheet" href="../../../../../public/assets/bootstrap/dist/css/bootstrap.min.css">
+    </head>
+    
+    <title> <?php echo xlt("ClaimRev Connect - Setup"); ?></title>
+
+
+<body>
+    <div class="row"> 
+        <div class="col">
+            <?php
+                require '../templates/navbar.php';
+            ?>
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col">
+            <h1><?php echo xlt("Setup"); ?></h1>        
+        </div>
+    </div>
+    <div class="row"> 
+        <div class="col-6">
+            <div class="card">
+                <ul>
+                    <li>
+                        <h6><?php echo xlt("x12 Partner Record"); ?></h6>
+                        <?php
+                        if (ClaimRevModuleSetup::doesPartnerExists()) {
+                            echo xlt("It looks like your X12 partner record is setup.");
+                        } else {
+                            echo xlt("Your x12 Partner has not been created, please contact us if you need assistance.");
+                        }
+                        ?>
+                    </li>
+                    <li>
+                        <h6>
+                            <?php echo xlt("Background Services"); ?>
+                            
+                        </h6>
+                        <?php echo xlt("There are required background services that are needed to send claims, pick up reports, and check eligibility. They are listed below in a table, but if there is something strange going on use the button to re-create the records."); ?>
+                        
+                        <form method="post" action="setup.php">
```

Comment:
csrf line 3a (this will go into 3 places):
```
<input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken('ClaimRevModule')); ?>" />
```